#!/usr/bin/env python
# _*_ coding:utf-8 _*_

'''
  Description:
    人脸跟踪时arduino节点的可以调用dynamic_reconfigure动态更新参数.
'''

from dynamic_reconfigure.parameter_generator_catkin import *

PACKAGE = "arduino_servo_connect"
gen = ParameterGenerator()

gen.add("firmware_pub_rate", double_t, 0, "pub arduino firmware ver rate", 0.1, 0.01, 1.0)
gen.add("default_x_deg", int_t, 0, "servo x default degree", 90, 0, 180)
gen.add("default_y_deg", int_t, 0, "servo y default degree", 0, 0, 180)
gen.add("servo_move_delay", int_t, 0, "when exit move servo delay", 60, 0, 150)

exit(gen.generate(PACKAGE, "arduino_dynamic_reconfig", "arduino_servo"))
